README.md

```diff
@@ -38,7 +37,41 @@ There are two steps for uploading docs:
    --help                     Show this message and exit.
    ```
 
+   docuploader will take the documentation in `DOCUMENTATION_PATH`,
+   `docs.metadata` or `docs.metadata.json` and create a tarball (`tgz`), then
+   send the tarball to the bucket specified in `--staging-bucket` option with a
+   prefix specified in `--destination-prefix`.
+
 `docuploader` can use ADC (Application Default Credentials) or a given service account. To use ADC, run `gcloud auth application-default login` prior to upload to gain credentials.
 
 For an example of using `docuploader`, see
 [example usage in googleapis/google-cloud-go](https://github.com/googleapis/google-cloud-go/blob/main/internal/kokoro/publish_docs.sh).
+
+## Requirements for DocFX YAML tarballs
+
+The tarballs containing DocFX YAML files must adhere to the following requirements:
+
+1. The tarball must contain metadata in the form of the `Metadata` message from
+   [`metadata.proto`](./docuploader/protos/metadata.proto). This can be
+   represented in protobuf text format in a file called `docs.metadata`, or in
+   protobuf JSON format in a file called `docs.metadata.json`.
+    * [`docs.metadata` example](https://github.com/googleapis/doc-pipeline/blob/29b25c0399aa295db9dfa59fa78214c9a810350d/testdata/python/docs.metadata)
```

Comment:
Would these be better as head references rather than a specific commit? (If metadata.proto changes, presumably the examples will be updated, but this commit will still be old.)

Reply:
Fair point, only concern with using the head reference would be if things were to change on the other repo. But, for a working example I think they should be up to date and potentially break the link (but easily fixable), rather than always have a valid link but show an older example which may not work at the time. Updated the links.